components/Nav.js

```diff
@@ -17,7 +17,7 @@ const MainNav = () => {
       <Navbar.Toggle aria-controls="basic-navbar-nav" />
       <Navbar.Collapse id="basic-navbar-nav">
         <Nav className="ml-auto">
-          <Nav.Link href="">About</Nav.Link>
+          <Nav.Link href="/about">About</Nav.Link>
```

Comment:
Is there a way to integrate Next.js Link into this?

Cuz rn, the page is doing a full reload every time you click that link

https://nextjs.org/docs/api-reference/next/link

Reply:
if you can't find an easy fix, you should make an issue and we can do that in the next sprint cycle

Reply:
@jimmyygit did the nav I think